src/controllers/expenseController.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+
+const expenseServices = require('../services/expenseServices');
+const userServices = require('../services/userServices');
+
+function getAllExpenses(req, res) {
```

Comment:
Use arrow functions instead of regular function expressions for consistency and shorter syntax.